frontend/src/bundles/auth/components/sign-up-form/constants/constants.ts

```diff
@@ -3,6 +3,7 @@ import { type UserSignUpRequestDto } from '~/bundles/users/users';
 const DEFAULT_SIGN_UP_PAYLOAD: UserSignUpRequestDto = {
```

Comment:
I think you should create custom type/interface for this case

Reply:
Why do you need this constant?